src/wazuh_modules/vulnerability_scanner/tests/unit/databaseFeedManager_test.cpp

```diff
@@ -102,6 +100,25 @@ const std::string VULNERABILITY_CANDIDATE_EXAMPLE = R"(
 const std::string CNA_NAME {"cnaName"};
 const std::string PACKAGE_NAME {"libmagic-mgc"};
 const std::string CORRUPTED_PACKAGE_NAME {"libcorrupted"};
+const nlohmann::json CNA_MAPPINGS = "\
+    { \
+      \"cnaMapping\": { \
+        \"alas\": \"alas_$(MAJOR_VERSION)\", \
+        \"alma\": \"alma_$(MAJOR_VERSION)\", \
+        \"redhat\": \"redhat_$(MAJOR_VERSION)\", \
+        \"suse\": \"$(PLATFORM)_$(MAJOR_VERSION)\" \
+      }, \
+      \"majorVersionEquivalence\": { \
+        \"amzn\": { \
+          \"2018\": \"1\" \
+        } \
+      }, \
+      \"platformEquivalence\": { \
+        \"sled\": \"suse_desktop\", \
+        \"sles\": \"suse_server\" \
+      } \
+    } \
+    "_json;
```

Comment:
This will achieve the same result
```suggestion
const nlohmann::json CNA_MAPPINGS = R"***(
{
  "cnaMapping": {
    "alas": "alas_$(MAJOR_VERSION)",
    "alma": "alma_$(MAJOR_VERSION)",
    "redhat": "redhat_$(MAJOR_VERSION)",
    "suse": "$(PLATFORM)_$(MAJOR_VERSION)"
  },
  "majorVersionEquivalence": {
    "amzn": {
      "2018": "1"
    }
  },
  "platformEquivalence": {
    "sled": "suse_desktop",
    "sles": "suse_server"
  }
}
)***"_json;
```